features/google/src/components/importExplanationExpandable/importExplanationExpandable.jsx

```diff
@@ -202,14 +206,17 @@ const ImportExplanationExpandable = ({
                     )}
                 </div>
                 <PolyButton
-                    className="btn primary"
+                    className="bg-red"
```

Comment:
if you take a look at the Facebook Import, you will see that this button is not red, it would be `type="outline"`.